docs/intro/tutorial02.md

```diff
@@ -131,6 +131,61 @@ $ wasm-to-oci push demo.wasm mycontainerregistry007.azurecr.io/krustlet-tutorial
 1](tutorial01.md) of this tutorial. If you are publishing the Rust example, use
 `target/wasm32-wasi/debug/demo.wasm` instead.
 
+## Create a container registry pull secret 
+
+Unless your container registry is enabled with anonymous access, you need to authenticate krustlet to pull images from it. At the moment, there is no flag in the Azure portal to make a registry public, but you can [create a support ticket](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-faq#how-do-i-enable-anonymous-pull-access) to have it enabled manually. 
+
+Without public access to the container registry, you need to create a _Kubernetes pull secret_. The steps below for Azure are [extracted from the Azure documentation](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-auth-kubernetes), and repeated herefor convenience.
```

Comment:
s/herefor/here for/

Reply:
Will do

Reply:
Done, some http-links still remain long.

Reply:
HTTP links should be just fine!